Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -717,13 +717,11 @@ String AccessibilityRenderObject::textUnderElement(AccessibilityTextUnderElement
         if (WeakPtr renderText = dynamicDowncast<RenderText>(*m_renderer)) {
             if (WeakPtr renderTextFragment = dynamicDowncast<RenderTextFragment>(*renderText)) {
                 // The alt attribute may be set on a text fragment through CSS, which should be honored.
-                const auto& altText = renderTextFragment->altText();
-                if (!altText.isEmpty())
+                if (auto& altText = renderTextFragment->altText(); !altText.isNull())
```

Comment:
I'm not 100% sure, but given `String` has its negation operator defined as follows:

```
inline bool operator!(const String& string) { return string.isNull(); }
```

Maybe you don't need the second clause in that if statement?

Reply:
I'm going to leave this explicit for now. But happy to fix as a follow-up.